department-of-veterans-affairs/va.gov-cms
# VACMS-2797: Add new tests to check for PHP notices, warnings & errors.

## Description
This adds two new tests to check for PHP WARNINGs and ERRORs. I couldn't check for both so just made two new tests. 
#2797 

The first test run of this should fail, as we still have some existing code introducing an error in test runs. 

## Testing done
Locally tested on Lando. 

## Screenshots


## QA steps
1. Create a PHP warning or error in code so it shows in the logs (normally a full test run might do this)
1. Run `lando yaml-tests php-`
1. Respective test should fail.
